### PR TITLE
code sdr

### PR DESCRIPTION
**Pull request by ravinab29**
code of sand drawing robot

### CODE REVIEW

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@ravinab29 
All the required files have to be uploaded as mentioned in the documentation guideline.
1. Refer guideline and upload Images, Videos, Solidworks files, Report, Poster in respective folders.
2. Python/Arduino codes have to be inside the src folder.
3. Maintain general terminology to name your files to make it comprehensible by all kinds of audiences. 
4. README.md file is empty. Refer guideline and update it accordingly.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,80 @@
 # Sand-Drawing-Robot
-Srishti 2020
+**Srishti 2020**
+## Abstract 
+The aim of this project is to draw a clear and neat structure of any image in sand given with maximum accuracy level.
+ 
+ ![image](C:\Users\hp\Documents\sdr.jpg)
```
Comment (agshubhs) [inline README.md]: Image is not rendering.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,80 @@
...
+
+**In any malls or tourism places** - we can draw pictures on sand instead of paper
+## Limitations:
+Time taken to draw a picture is very high
+## Future Improvements:
+We can use more servo motors to draw a given picture so that the time taken will be less.
+We can also increase the roatation of servo motor and decrease the delay to speeden the drawing.
+
+## Team members:
+
```
Comment (agshubhs) [inline README.md:82]: Should be in ordered list

**Comment by agshubhs** [comment]
@ravinab29 
1. Mechanical Design Folder is missing.
2.  Poster and Report, Images and Videos folder are not there.
3.  Readme is not upto the  mark.
4. Flow chart is missing.

**Comment by ravinab29** [comment]
sir i have updated the readme part. kindly review it.

**Comment by SanjeevKrishnan** [comment]
> @ravinab29
> 
> 1. Mechanical Design Folder is missing.
> 2. Poster and Report, Images and Videos folder are not there.
> 3. Readme is not upto the  mark.
> 4. Flow chart is missing.

@ravinab29  Still these changes are not updated. Also, tag us when all the changes are done. Reviewing individual files is not feasible.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,80 @@
...
+2. 1 servo motor
+3. 1 Arduino mega
+4. Jumper wires 
+5. 4 linear motion bearings
+6. 1 6mm timing belt
+7. 2 20teeth pulley 5mm bore
+8. 2mm and 3mm nuts and bolts
+
+## Workflow:
+User gives an image, python code converts image to pixelated image such that only two pixel values will be stored for the image ,one stepper motor moves the shaft vertically with the given time gap and another stepper motor makes  the servo motor slide on the shaft, servo motor turns the stick up and down according to the given pixels of the image, with the lines drawn by the stick required image is drawn.
```
Comment (SanjeevKrishnan) [inline README.md]: Flowchart is missing

File: README.md [hunk 4]
```diff
@@ -1,2 +1,80 @@
...
+-----------| ---------  
+2 Timing belt and pulleys | 910   
+4 Linear bearings | 948     
+Wooden cardboard | 200     
+8mm brass rods | 640     
+Nuts, bolts,jumper wires | 250     
+Miscellaneous | 200     
+Total | 3148       
+## Applications:
+**Beaches** - on sand we can draw one’s picture.
```
Comment (SanjeevKrishnan) [inline README.md]: Applications, Limitations, and Future improvements have to be unordered list.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,80 @@
...
+Time taken to draw a picture is very high
+## Future Improvements:
+We can use more servo motors to draw a given picture so that the time taken will be less.
+We can also increase the roatation of servo motor and decrease the delay to speeden the drawing.
+
+## Team members:
+                        
+
+ 
+ [Sejal Agarwal](https://github.com/sejal-ag)
```
Comment (SanjeevKrishnan) [inline README.md]: Team Members and Mentors names should be in ordered list.

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@ravinab29 Changes are still required!
1. Videos is missing. Try to have at least one video as it will be advantageous to your repo. You can try making a screen recorded video of Arduino stimulation.
2. Rename "images" as "Images"
3. Mechanical Design folder is missing.
4. Justify the text in Report.
5. File names are not proper. Names should be self-explanatory.
6. In README.md, justify text using `<p align="justify">Content<\p>` where content is the text you want to justify.

**Comment by agshubhs** [comment]
@ravinab29 
1. I should be capital in Images folde

**Comment by agshubhs** [comment]
@ravinab29  Nice work. PR looks good to me.

**Pull request merged**